packages/ruleset/src/functions/resource-response-consistency.js

```diff
@@ -8,6 +8,7 @@ const { isObject } = require('@ibm-cloud/openapi-ruleset-utilities');
 const {
   getResourceSpecificSiblingPath,
   getResponseCodes,
+  getSuccessResponseSchemaForOperation,
```

Comment:
I needed a stretch of code that already existed in this function so I moved it out to a utility